<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>basic</class>
 <widget class="QWidget" name="basic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="btn_select">
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="5" colspan="2">
    <widget class="QScrollArea" name="scroll_plugin">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="plugin_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>150</width>
        <height>272</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="matplotlib">
         <property name="text">
          <string>matplotlib</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="multiprocessing">
         <property name="text">
          <string>multiprocessing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="numpy">
         <property name="text">
          <string>numpy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pyqt5">
         <property name="text">
          <string>pyqt5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pyqt6">
         <property name="text">
          <string>pyqt6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pyside2">
         <property name="text">
          <string>pyside2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pyside6">
         <property name="text">
          <string>pyside6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tensorflow">
         <property name="text">
          <string>tensorflow</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tk_inter">
         <property name="text">
          <string>tk-inter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="torch">
         <property name="text">
          <string>torch</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="checkbox_onefile">
     <property name="text">
      <string>单文件</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="checkbox_disable_console">
     <property name="text">
      <string>禁用命令行</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="checkbox_standalone">
     <property name="text">
      <string>包含运行环境</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="checkbox_require_admin">
     <property name="text">
      <string>请求管理员权限</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="btn_start">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>打包</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label_path">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
